rest_docs/rest-catalog-open-api.yaml

```diff
@@ -1381,15 +1381,32 @@ components:
               } }
 
     CreateNamespaceResponse:
-      description: A succesful call to create a namespace
+      description:
+        Represents a successful call to create a namespace.
+        Returns the namespace created, as well as any properties that were stored for the namespace,
+        including those the server might have added. Implementations are not required to support namespace
+        properties.
       content:
         application/json:
           schema:
             type: object
+            required:
+              - namespace
             properties:
-              created:
-                type: boolean
-                description: true if the namespace was added to the catalog
+              namespace:
+                $ref: '#/components/schemas/Namespace'
+              properties:
+                type: object
+                additionalProperties:
+                  type: string
+                description:
+                  Properties stored on the namespace, if supported by the server.
+                example: { "owner": "Ralph", "created_at": "1452120468" }
+                default: { }
```

Comment:
What does default mean? Is this what you should fill in as the default if it is missing, or what the server should send as a default?